### ISSUE TITLE
hack does not compile ....... .  

### ISSUE THREAD

**Issue by duxeyo** [bidengaming/rosehacc#1]
hi i am tryed complication the hacking and it say "unresolved external symbol Direct3DCreate9" please help me in dsicord

**Comment by posh** [comment]
yes error this i have seen it before... I think google.com.au held answer for me...

**Comment by SpringoMode** [comment]
Judging by your github profile you seem like a skid. So now I will not help you 👍

**Comment by bidengaming** [comment]
amm maybe u is try amm wild thought amm reinstalling visual studios code ? 👍

**Comment by cheggu** [comment]
error could not find include windows.h ??? help polska

**Comment by SpringoMode** [comment]
Use windows...
